FreshRSS/FreshRSS
# Fix/Improved: Slider mobile

Part 1: Regression from #4394 (The mobile view was broken (not 100% width, close button not visible in the slider))
Part 2: slider loads only the content, not the whole layout (introduced with #4310)


How to test the feature manually:

1. open a slider (f.e. config a feed)
2. see this slider in desktop mode
3. see this slider in mobile mode

Pull request checklist:
